frontend/pages/index.js

```diff
@@ -44,20 +44,20 @@ const darkTheme = createTheme({
 
 OpenAPI.BASE = "http://localhost:8000";
 
-export default function Home() {
-
+const contract = {}; // require("../../artifacts/contracts/AINFT.sol/SyntheticDreams.json");
```

Comment:
@Sandeepsuresh1998 this file was missing and also should not be required on the render of a component.